docs/manual/format.md

```diff
@@ -24,16 +24,19 @@ package file is divided in 4 logical sections:
 ```
 
 All 2 and 4 byte "integer" quantities (int16 and int32) are stored in
```

Comment:
Sorry, forgot to mention this yesterday: we also have 64bit integers where this also applies.